test.py

```diff
@@ -0,0 +1,4 @@
+import flask
+print("hello world")
+print("hello prbot")
```

Comment:
**Suggestion:** Avoid using print statements for debugging. Consider using a logger instead.
```suggestion
import logging
logging.basicConfig(level=logging.DEBUG)
logging.debug("hello world")
logging.debug("hello prbot")
```